examples/simple_trainer.py

```diff
@@ -150,22 +156,32 @@ def adjust_steps(self, factor: float):
 
 
 def create_splats_with_optimizers(
-    points: Tensor,  # [N, 3]
-    rgbs: Tensor,  # [N, 3]
+    parser: Parser,
+    init_type: str = "sfm",
+    init_num_pts: int = 100_000,
+    init_opacity: float = 0.1,
+    init_scale: float = 1.0,
     scene_scale: float = 1.0,
     sh_degree: int = 3,
-    init_opacity: float = 0.1,
     sparse_grad: bool = False,
     batch_size: int = 1,
     feature_dim: Optional[int] = None,
     device: str = "cuda",
 ) -> Tuple[torch.nn.ParameterDict, torch.optim.Optimizer]:
-    N = points.shape[0]
+    if init_type == "sfm":
+        points = torch.from_numpy(parser.points).float()
+        rgbs = torch.from_numpy(parser.points_rgb / 255.0).float()
+    elif init_type == "random":
+        points = scene_scale * (torch.rand((init_num_pts, 3)) * 3 * 2 - 3)
```

Comment:
What is the reason for the `* 3` here? Is it for the nerf synthetic data that mostly lives in the range of [-1.5, 1.5]? Also does it work with COLMAP captures? Maybe yes to most of the captures because we normalize the scene. But as the scale normalization was based on camera distribution, it is not guaranteed that the scene lives in [-1.0, 1.0]. So I think we should have a scaler here to control the initialized range, which can be an argument with default value as 3. In fact, I thought that `init_scale` is for this purpose, but its not. Should we use `init_scale` here instead of multiplying it to the `dist_avg`?

Reply:
`* 3` specifies the extent of the randomized initialized points. The paper recommends 3x the camera extent but also says there is not a big difference between 1x and 3x. 1x would make more sense for synthetic scenes. Created variable `init_extent`.

Reply:
Assume `scene_scale` ~= the furthest camera in the COLMAP capture and the camera poses are centered around zero, I think this would work.